TP-1/exercice-avancé/test/test_temp_controller.c

```diff
@@ -3,38 +3,37 @@
 #include "mock_temp_sensor.h"
 
 void setUp(void) {
-    // Initialisez les mocks "Bonne pratique"
-    // ??
+    mock_temp_sensor_Init();
 }
 
 void tearDown(void) {
-    // Nettoiez les mocks "Bonne pratique"
-    // ??
+    mock_temp_sensor_Verify();
+    mock_temp_sensor_Destroy();
 }
 
 void test_check_temperature_should_return_1_when_too_hot(void) {
-    read_temperature_sensor_ExpectAndReturn(/* Température élevée */);
-    TEST_ASSERT_EQUAL(/* Résultat attendu */, check_temperature());
+    read_temperature_sensor_ExpectAndReturn(80);
+    TEST_ASSERT_EQUAL(1, check_temperature());
 }
 
 void test_check_temperature_should_return_minus_1_when_too_cold(void) {
-    read_temperature_sensor_ExpectAndReturn(/* Température basse */);
-    TEST_ASSERT_EQUAL(/* Résultat attendu */, check_temperature());
+    read_temperature_sensor_ExpectAndReturn(-10);
+    TEST_ASSERT_EQUAL(-1, check_temperature());
 }
 
 void test_check_temperature_should_return_0_when_normal(void) {
-    read_temperature_sensor_ExpectAndReturn(/* Température normale */);
-    TEST_ASSERT_EQUAL(/* Résultat attendu */, check_temperature());
+    read_temperature_sensor_ExpectAndReturn(25);
+    TEST_ASSERT_EQUAL(0, check_temperature());
 }
 
 
 // Testes supplémentaires : Cas limites exacts
 void test_check_temperature_should_handle_exact_limits(void) {
-    read_temperature_sensor_ExpectAndReturn(/* ?? */); // Limite basse
-    TEST_ASSERT_EQUAL(/* ?? */, check_temperature());
+    read_temperature_sensor_ExpectAndReturn(0); // Limite basse
+    TEST_ASSERT_EQUAL(0, check_temperature());
 
-    read_temperature_sensor_ExpectAndReturn(/* ?? */); // Limite haute
-    TEST_ASSERT_EQUAL(/* ?? */, check_temperature());
+    read_temperature_sensor_ExpectAndReturn(50); // Limite haute
+    TEST_ASSERT_EQUAL(0, check_temperature());
 
```

Comment:
Refer back to "TP-1/advanced-exercise/src/temp_controller.c" to check the limit values.
And the assertion is wrong anyways as shown in the /results